doc/lightning-txprepare.7.md

```diff
@@ -6,17 +6,35 @@ internal wallet.
 SYNOPSIS
 --------
 
-**txprepare** *destination* *satoshi* \[*feerate*\] \[*minconf*\]
+**txprepare** *output* \[*feerate*\] \[*minconf*\]
 
 DESCRIPTION
 -----------
 
 The **txprepare** RPC command creates an unsigned transaction which
-spends funds from c-lightning’s internal wallet to the address specified
-in *destination*.
-
-Effectively, it is the first part of a **withdraw** command, and uses
-the same parameters. The second part is provided by **txsend**.
+spends funds from c-lightning’s internal wallet to the output specified
+in *output*.
+
+The *output* is the array of outputs that must specify *destination*
+and *satoshi*. Its format is like:
+\[\{*destination*: address1, *satoshi*: satoshi1\},
```

Comment:
Couple of gripes with the format of this

- The bitcoin-cli interface for outputs is [{"address":"amount"}]. Ex: `[{"bcrt1qc80w6na6rhvh9jtjgv9wwjlm8crxfkcfg87gxq":"20000sat"}]`. It seems preferable to mimic this, if possible.
- If not, I think we should rename `satoshi` to `amount`. It's a bit confusing that you can specify the `satoshi` field as `2.999btc`.

Reply:
> The bitcoin-cli interface for outputs is [{"address":"amount"}]. Ex: [{"bcrt1qc80w6na6rhvh9jtjgv9wwjlm8crxfkcfg87gxq":"20000sat"}]. It seems preferable to mimic this, if possible.

Thank you, I'll try the `[{"address":"amount"}]`.
> If not, I think we should rename satoshi to amount. It's a bit confusing that you can specify the satoshi field as 2.999btc.

Yes, `amount` is better.
BTW, should `fundchannel` and `fundchannel_start` also use `amount` to replace `satoshi`?

Reply:
Sorry for blocking so long time...now I changes the format like [{address: amount}]. What do you think about it now?